cmd/erpc/main.go

```diff
@@ -30,16 +32,53 @@ func init() {
 func main() {
 	logger := log.With().Logger()
 
-	logger.Info().Msgf("starting eRPC version: %s, commit: %s", common.ErpcVersion, common.ErpcCommitSha)
+	// Define the flag for the config file
+	configFileFlag := &cli.StringFlag{
+		Name:     "config",
+		Usage:    "Config file to use (by default checking erpc.js, erpc.ts, erpc.yaml, erpc.yml)",
+		Required: false,
+	}
```

Comment:
To be backward compatible I think we should support arg-mode as well, like:
```
erpc myconf.yml
```
Perhaps we should deprecate it though on next 'major' release.